2021-06-08 22:47:59,330 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59813 failed: java.net.SocketException: Connection reset
2021-06-08 22:47:59,332 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59c505a904ae6StandardJMSBroker#4 and tcp:///192.168.178.21:59810@59812 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 22:48:09,643 INFO [n-59c505a904ae6.a-59c50ad39caa1.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-224809_350590939.txt
2021-06-08 22:48:09,649 ERROR [n-59c505a904ae6.a-59c50ad39caa1.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-224809_350590939.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:48:43,473 WARN [n-59c505a904ae6.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 22:48:43,474 WARN [n-59c505a904ae6.IDirectory] - 	df@n-59c52cc6915b3
java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:47:56,668 ERROR [n-59c52cc6915b3.a-59c5318a4996b.SimpleExecutionCycle] - Error when executing bean 'WorkerBean'
java.lang.NullPointerException: Cannot read field "position" because "this.currentOrder" is null
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.handleCheckDistance(WorkerBean.java:207)
	at de.dailab.jiactng.aot.gridworld.client.WorkerBean.execute(WorkerBean.java:94)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:54:30,013 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8884 mb of usable space - resetting to maximum available disk space: 8884 mb
2021-06-08 22:54:35,483 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8884 mb of usable space - resetting to maximum available disk space: 8884 mb
2021-06-08 22:54:35,774 INFO [n-59cb1b2876dd2.a-59cb1b4395a6d.BrokerBean] - starting broker agent
2021-06-08 22:54:35,784 INFO [n-59cb1b2876dd2.a-59cb1a4ecee19.WorkerBean] - Starting worker agent with id: a-59cb1a4ecee19
2021-06-08 22:54:35,785 INFO [n-59cb1b2876dd2.a-59cb1a0bf35f4.WorkerBean] - Starting worker agent with id: a-59cb1a0bf35f4
2021-06-08 22:54:35,789 INFO [n-59cb1b2876dd2.a-59cb181e4d663.WorkerBean] - Starting worker agent with id: a-59cb181e4d663
2021-06-08 22:54:35,858 WARN [n-59cb1b2876dd2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:36,863 WARN [n-59cb1b2876dd2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:37,868 WARN [n-59cb1b2876dd2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:44,844 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59845 failed: java.net.SocketException: Connection reset
2021-06-08 22:54:44,847 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cb06d3120daStandardJMSBroker#4 and tcp:///192.168.178.21:59842@59844 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 22:54:50,336 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8884 mb of usable space - resetting to maximum available disk space: 8884 mb
2021-06-08 22:54:50,707 INFO [n-59cb4fb95443e.a-59cb4f4d0a3d5.BrokerBean] - starting broker agent
2021-06-08 22:54:50,721 INFO [n-59cb4fb95443e.a-59cb5463ba431.WorkerBean] - Starting worker agent with id: a-59cb5463ba431
2021-06-08 22:54:50,728 INFO [n-59cb4fb95443e.a-59cb540ccf23c.WorkerBean] - Starting worker agent with id: a-59cb540ccf23c
2021-06-08 22:54:50,730 INFO [n-59cb4fb95443e.a-59cb553463dcc.WorkerBean] - Starting worker agent with id: a-59cb553463dcc
2021-06-08 22:54:50,801 WARN [n-59cb4fb95443e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:51,806 WARN [n-59cb4fb95443e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:52,807 WARN [n-59cb4fb95443e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:54:53,550 INFO [n-59cb06d3120da.a-59cb06b5b0059.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-225453_532795932.txt
2021-06-08 22:54:53,556 ERROR [n-59cb06d3120da.a-59cb06b5b0059.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-225453_532795932.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:54:56,833 ERROR [n-59cb4fb95443e.a-59cb4f4d0a3d5.SimpleExecutionCycle] - Error when executing bean 'BrokerBean'
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.execute(BrokerBean.java:115)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:54:59,484 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cb06d3120daStandardJMSBroker#6 and tcp:///192.168.178.21:59851@59853 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 22:54:59,484 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59854 failed: java.net.SocketException: Connection reset
2021-06-08 22:55:08,612 INFO [n-59cb06d3120da.a-59cb06b5b0059.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-225508_258864927.txt
2021-06-08 22:55:08,616 ERROR [n-59cb06d3120da.a-59cb06b5b0059.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-225508_258864927.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:55:18,300 WARN [n-59cb06d3120da.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 22:55:18,301 WARN [n-59cb06d3120da.IDirectory] - 	df@n-59cb1b2876dd2
2021-06-08 22:55:33,300 WARN [n-59cb06d3120da.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 22:55:33,301 WARN [n-59cb06d3120da.IDirectory] - 	df@n-59cb4fb95443e
2021-06-08 22:57:08,152 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8887 mb of usable space - resetting to maximum available disk space: 8887 mb
2021-06-08 22:57:14,358 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8886 mb of usable space - resetting to maximum available disk space: 8886 mb
2021-06-08 22:57:14,753 INFO [n-59cd67ba430e2.a-59cd673c87ac5.BrokerBean] - starting broker agent
2021-06-08 22:57:14,782 INFO [n-59cd67ba430e2.a-59cd6ca21de9d.WorkerBean] - Starting worker agent with id: a-59cd6ca21de9d
2021-06-08 22:57:14,796 INFO [n-59cd67ba430e2.a-59cd6c4d8a028.WorkerBean] - Starting worker agent with id: a-59cd6c4d8a028
2021-06-08 22:57:14,798 INFO [n-59cd67ba430e2.a-59cd6d74c39e8.WorkerBean] - Starting worker agent with id: a-59cd6d74c39e8
2021-06-08 22:57:14,816 WARN [n-59cd67ba430e2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:57:15,821 WARN [n-59cd67ba430e2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:57:16,829 WARN [n-59cd67ba430e2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:57:20,862 ERROR [n-59cd67ba430e2.a-59cd673c87ac5.SimpleExecutionCycle] - Error when executing bean 'BrokerBean'
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.execute(BrokerBean.java:116)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:57:22,753 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59936 failed: java.net.SocketException: Connection reset
2021-06-08 22:57:22,756 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cd4c3bf227eStandardJMSBroker#4 and tcp:///192.168.178.21:59931@59934 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 22:57:32,748 INFO [n-59cd4c3bf227e.a-59cd5151dbda1.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-225732_765336255.txt
2021-06-08 22:57:32,749 ERROR [n-59cd4c3bf227e.a-59cd5151dbda1.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-225732_765336255.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:57:56,532 WARN [n-59cd4c3bf227e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 22:57:56,532 WARN [n-59cd4c3bf227e.IDirectory] - 	df@n-59cd67ba430e2
2021-06-08 22:59:37,572 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8877 mb of usable space - resetting to maximum available disk space: 8877 mb
2021-06-08 22:59:48,287 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8877 mb of usable space - resetting to maximum available disk space: 8877 mb
2021-06-08 22:59:48,690 INFO [n-59cfa5a007cf6.a-59cfa4f62b2fd.BrokerBean] - starting broker agent
2021-06-08 22:59:48,703 INFO [n-59cfa5a007cf6.a-59cfaa310187d.WorkerBean] - Starting worker agent with id: a-59cfaa310187d
2021-06-08 22:59:48,708 INFO [n-59cfa5a007cf6.a-59cfa9e546080.WorkerBean] - Starting worker agent with id: a-59cfa9e546080
2021-06-08 22:59:48,710 INFO [n-59cfa5a007cf6.a-59cfab0c9c668.WorkerBean] - Starting worker agent with id: a-59cfab0c9c668
2021-06-08 22:59:48,720 WARN [n-59cfa5a007cf6.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:59:49,720 WARN [n-59cfa5a007cf6.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:59:50,720 WARN [n-59cfa5a007cf6.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:59:51,720 WARN [n-59cfa5a007cf6.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 22:59:55,740 ERROR [n-59cfa5a007cf6.a-59cfa4f62b2fd.SimpleExecutionCycle] - Error when executing bean 'BrokerBean'
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.execute(BrokerBean.java:116)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 22:59:57,939 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59955 failed: java.net.SocketException: Connection reset
2021-06-08 22:59:57,941 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#4 and tcp:///192.168.178.21:59952@59954 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:00:07,211 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230007_1119936114.txt
2021-06-08 23:00:07,218 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230007_1119936114.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:00:40,885 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:00:40,885 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59cfa5a007cf6
2021-06-08 23:01:41,680 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8883 mb of usable space - resetting to maximum available disk space: 8883 mb
2021-06-08 23:01:42,016 INFO [n-59d14c6f85e4a.a-59d14bd3eeee9.BrokerBean] - starting broker agent
2021-06-08 23:01:42,027 INFO [n-59d14c6f85e4a.a-59d150d818c45.WorkerBean] - Starting worker agent with id: a-59d150d818c45
2021-06-08 23:01:42,032 INFO [n-59d14c6f85e4a.a-59d1508179b1c.WorkerBean] - Starting worker agent with id: a-59d1508179b1c
2021-06-08 23:01:42,034 INFO [n-59d14c6f85e4a.a-59d151a8a56fc.WorkerBean] - Starting worker agent with id: a-59d151a8a56fc
2021-06-08 23:01:42,167 WARN [n-59d14c6f85e4a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:43,163 WARN [n-59d14c6f85e4a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:44,167 WARN [n-59d14c6f85e4a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:50,617 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#6 and tcp:///192.168.178.21:59961@59963 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:01:50,618 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59964 failed: java.net.SocketException: Connection reset
2021-06-08 23:01:53,995 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8883 mb of usable space - resetting to maximum available disk space: 8883 mb
2021-06-08 23:01:54,406 INFO [n-59d17a5bb4d3e.a-59d1798dfdaa1.BrokerBean] - starting broker agent
2021-06-08 23:01:54,415 INFO [n-59d17a5bb4d3e.a-59d17e7f45969.WorkerBean] - Starting worker agent with id: a-59d17e7f45969
2021-06-08 23:01:54,419 INFO [n-59d17a5bb4d3e.a-59d17e29356f8.WorkerBean] - Starting worker agent with id: a-59d17e29356f8
2021-06-08 23:01:54,420 INFO [n-59d17a5bb4d3e.a-59d17f5033928.WorkerBean] - Starting worker agent with id: a-59d17f5033928
2021-06-08 23:01:54,430 WARN [n-59d17a5bb4d3e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:55,436 WARN [n-59d17a5bb4d3e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:56,436 WARN [n-59d17a5bb4d3e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:57,439 WARN [n-59d17a5bb4d3e.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:01:59,850 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230159_1254367998.txt
2021-06-08 23:01:59,852 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230159_1254367998.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:02:01,465 ERROR [n-59d17a5bb4d3e.a-59d1798dfdaa1.SimpleExecutionCycle] - Error when executing bean 'BrokerBean'
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.Map.get(Object)" is null
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.execute(BrokerBean.java:116)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:02:03,765 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#8 and tcp:///192.168.178.21:59968@59970 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:02:03,766 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59971 failed: java.net.SocketException: Connection reset
2021-06-08 23:02:12,917 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230212_762340172.txt
2021-06-08 23:02:12,921 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230212_762340172.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:02:25,915 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:02:25,920 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d14c6f85e4a
2021-06-08 23:02:40,915 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:02:40,916 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d17a5bb4d3e
2021-06-08 23:03:32,061 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8882 mb of usable space - resetting to maximum available disk space: 8882 mb
2021-06-08 23:03:32,381 INFO [n-59d2e76c72f9a.a-59d2e6cb76405.BrokerBean] - starting broker agent
2021-06-08 23:03:32,392 INFO [n-59d2e76c72f9a.a-59d2ebd9b88f1.WorkerBean] - Starting worker agent with id: a-59d2ebd9b88f1
2021-06-08 23:03:32,399 INFO [n-59d2e76c72f9a.a-59d2eb8318440.WorkerBean] - Starting worker agent with id: a-59d2eb8318440
2021-06-08 23:03:32,401 INFO [n-59d2e76c72f9a.a-59d2ecaa62f70.WorkerBean] - Starting worker agent with id: a-59d2ecaa62f70
2021-06-08 23:03:32,492 WARN [n-59d2e76c72f9a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:33,492 WARN [n-59d2e76c72f9a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:34,496 WARN [n-59d2e76c72f9a.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:49,328 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#10 and tcp:///192.168.178.21:59976@59978 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:03:49,328 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59979 failed: java.net.SocketException: Connection reset
2021-06-08 23:03:50,470 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230350_1580305300.txt
2021-06-08 23:03:50,471 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230350_1580305300.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:03:53,572 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8882 mb of usable space - resetting to maximum available disk space: 8882 mb
2021-06-08 23:03:54,050 INFO [n-59d33776201aa.a-59d336d79c4a5.BrokerBean] - starting broker agent
2021-06-08 23:03:54,063 INFO [n-59d33776201aa.a-59d33be2787b1.WorkerBean] - Starting worker agent with id: a-59d33be2787b1
2021-06-08 23:03:54,068 INFO [n-59d33776201aa.a-59d33b9aa7660.WorkerBean] - Starting worker agent with id: a-59d33b9aa7660
2021-06-08 23:03:54,070 INFO [n-59d33776201aa.a-59d33cc1dd394.WorkerBean] - Starting worker agent with id: a-59d33cc1dd394
2021-06-08 23:03:54,079 WARN [n-59d33776201aa.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:55,077 WARN [n-59d33776201aa.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:56,079 WARN [n-59d33776201aa.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:03:57,085 WARN [n-59d33776201aa.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:09,242 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#12 and tcp:///192.168.178.21:59985@59987 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:04:09,242 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59988 failed: java.net.SocketException: Connection reset
2021-06-08 23:04:10,919 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:04:10,920 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d2e76c72f9a
2021-06-08 23:04:12,585 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230412_722546360.txt
2021-06-08 23:04:12,586 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230412_722546360.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:04:14,718 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8882 mb of usable space - resetting to maximum available disk space: 8882 mb
2021-06-08 23:04:15,005 INFO [n-59d3867ab8902.a-59d385a5245cd.BrokerBean] - starting broker agent
2021-06-08 23:04:15,014 INFO [n-59d3867ab8902.a-59d38a960d37d.WorkerBean] - Starting worker agent with id: a-59d38a960d37d
2021-06-08 23:04:15,017 INFO [n-59d3867ab8902.a-59d38a43455f8.WorkerBean] - Starting worker agent with id: a-59d38a43455f8
2021-06-08 23:04:15,019 INFO [n-59d3867ab8902.a-59d38b69e8dc4.WorkerBean] - Starting worker agent with id: a-59d38b69e8dc4
2021-06-08 23:04:15,103 WARN [n-59d3867ab8902.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:16,102 WARN [n-59d3867ab8902.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:17,104 WARN [n-59d3867ab8902.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:32,184 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#14 and tcp:///192.168.178.21:59992@59994 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:04:32,184 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:59995 failed: java.net.SocketException: Connection reset
2021-06-08 23:04:32,692 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230432_1643127098.txt
2021-06-08 23:04:32,693 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230432_1643127098.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:04:37,713 WARN [org.apache.activemq.broker.BrokerService] - Temporary Store limit is 51200 mb (current temporary store usage is 0 mb). The temporary data directory: C:\Users\sinam\IdeaProjects\aot34 only has 8882 mb of usable space - resetting to maximum available disk space: 8882 mb
2021-06-08 23:04:38,025 INFO [n-59d3dc1e7ecb2.a-59d3db7f2693d.BrokerBean] - starting broker agent
2021-06-08 23:04:38,035 INFO [n-59d3dc1e7ecb2.a-59d3e0734cc11.WorkerBean] - Starting worker agent with id: a-59d3e0734cc11
2021-06-08 23:04:38,039 INFO [n-59d3dc1e7ecb2.a-59d3e02235678.WorkerBean] - Starting worker agent with id: a-59d3e02235678
2021-06-08 23:04:38,040 INFO [n-59d3dc1e7ecb2.a-59d3e1494c90c.WorkerBean] - Starting worker agent with id: a-59d3e1494c90c
2021-06-08 23:04:38,155 WARN [n-59d3dc1e7ecb2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:39,156 WARN [n-59d3dc1e7ecb2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:40,159 WARN [n-59d3dc1e7ecb2.IDirectory] - Cannot find agent:
Agent:
 name='ServerAgent'
 aid=null
 owner=null
 state=null
 messageBoxAddress=null
 mobile=false

2021-06-08 23:04:40,920 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:04:40,922 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d33776201aa
2021-06-08 23:04:53,238 ERROR [n-59d3dc1e7ecb2.a-59d3db7f2693d.SimpleExecutionCycle] - Error when executing bean 'BrokerBean'
java.lang.NullPointerException: Cannot invoke "de.dailab.jiactng.agentcore.ontology.IAgentDescription.getMessageBoxAddress()" because "theWorker" is null
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.handleOrderCompleted(BrokerBean.java:325)
	at de.dailab.jiactng.aot.gridworld.client.BrokerBean.execute(BrokerBean.java:126)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.performExecutionCycleProcess(SimpleExecutionCycle.java:31)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle.processBeanExecution(BasicExecutionCycle.java:76)
	at de.dailab.jiactng.agentcore.execution.SimpleExecutionCycle.run(SimpleExecutionCycle.java:51)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:04:55,814 INFO [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Logging game C:\Users\sinam\IdeaProjects\aot34\logs\game_210608-230455_1443739815.txt
2021-06-08 23:04:55,815 ERROR [n-59cf81ba8401e.a-59cf804816020.ServerMetaBean] - Failed to log game results to file
java.nio.file.NoSuchFileException: logs\game_210608-230455_1443739815.txt
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:224)
	at java.base/java.nio.file.Files.write(Files.java:3556)
	at java.base/java.nio.file.Files.write(Files.java:3607)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.logGameResults(ServerBean.java:456)
	at de.dailab.jiactng.aot.gridworld.server.ServerBean.execute(ServerBean.java:376)
	at de.dailab.jiactng.agentcore.execution.BasicExecutionCycle$1.run(BasicExecutionCycle.java:79)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-08 23:04:55,920 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:04:55,922 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d3867ab8902
2021-06-08 23:04:57,726 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-59cf81ba8401eStandardJMSBroker#16 and tcp:///192.168.178.21:59999@60001 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-08 23:04:57,726 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.21:60002 failed: java.net.SocketException: Connection reset
2021-06-08 23:05:40,921 WARN [n-59cf81ba8401e.IDirectory] - Forget nodes due to missing alive message:

2021-06-08 23:05:40,922 WARN [n-59cf81ba8401e.IDirectory] - 	df@n-59d3dc1e7ecb2
